packages/content-conformance/src/content-file.ts

```diff
@@ -19,6 +20,17 @@ import type { Node } from 'unist'
 export class ContentFile extends VFile {
   __type = 'content' as const
 
+  // data: Record<string, any>
+
+  constructor(value: Compatible) {
+    super(value)
+    // @ts-expect-error - this.content is required by vfile-matter
+    // at compile-time but not at run-time
+    matter(this, {
+      strip: true,
```

Comment:
I'm wondering if I was wrong about this option. It might mess up the positional information in our errors (e.g. line count would be off by N, where N is the number of lines the frontmatter takes up).

Reply:
Hmm, yea it does mess with the positional information.

... could we _strip-and-replace_ with empty lines (as padding effectively)?

Reply:
@BRKalow, check this out: https://github.com/hashicorp/web-platform-packages/pull/118/commits/ccac4cb898e008c0645126d339cdf39e8fd3cc53

Reply:
Awesome! Can we abstract out the frontmatter parsing into a private method and call it in the constructor? Stylistic more than anything, but I think it'll help make the constructor really clear.

Reply:
Yep, great idea... on it 👍